index.html

```diff
@@ -0,0 +1,132 @@
+                    class="fab fa-linkedin-in"></i></a>
+            <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+            <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+        </div>
+    </section>
+    <section id="about">
+        <div class="container">
+            <h1>About Me</h1>
+            <div class="row">
+                <div class="col-md-6 text-center about-col">
+                    <div class="profile">
+                        <img src="images/picpic.jpg" class="user" alt="about image">
+                        <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from
+                            Vitebsk State University. I've got some basic understanding of c++ programming language but
+                            here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap,
+                            Javascript/Jquery.</p>
+                    </div>
+                </div>
+                <div class="col-md-6 skills-bar">
+                    <p>HTML</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 85%;">80%</div>
+                    </div>
+                    <p>CSS</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 70%;">60%</div>
+                    </div>
+                    <p>Javascript</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 40%;">40%</div>
+                    </div>
+                    <p>Wordpress</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 70%;">70%</div>
+                    </div>
+                    <p>GIT</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 30%;">50%</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="contact-btn"><a href="#">Contact me</a></div>
+    </section>
+    </main>
+    <footer class=" footer page-footer font-small blue pt-4">
+        <div class="container-fluid text-center text-md-left">
+            <div class="row">
+                <div class="col-md-6 mt-md-0 mt-3 lg">
+                    <h5 class="text-uppercase">Arinze's <span>Design</span></h5>
+                    <div class="footer-copyright text-left py-3">© 2020 Copyright
+                        <strong>ARINZE'S DESIGN</strong>
+                    </div>
+                </div>
+                <hr class="clearfix w-100 d-md-none pb-3">
+                <div class="col-md-3 sos-med">
+                    <div>
+                        <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i
```

Comment:
When you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.